lib/tok.js

```diff
@@ -342,13 +344,14 @@
 
     next: function(expressionStart){
       this.lastNewline = false;
-
       var toStream = this.options.saveTokens;
 
       do {
         var type = this.nextWhiteToken(expressionStart);
         if (toStream) {
-          var token = {type:type, value:this.getLastValue(), start:this.lastStart, stop:this.pos, white:this.tokens.length};
+          var lastValue = this.getLastValue();
```

Comment:
Since the last value is assigned to the `token`, you don't really need this since you're also passing on the token itself. You can simply read `token.value` to get these same value. This saves you the argument and extra changes here.